### DISCUSSION TITLE
我修改了python插件的值，为什么请求还是不变？

### DISCUSSION THREAD

**Discussion by xlyoung** [apache/apisix#7382] [General]
环境：
        apisix：2.14.1
            python 插件 0.2
         使用docker-compose启动，apisix 增加了python3环境，且加入了python插件持久化

## 脚本截图：

<img width="816" alt="image" src="https://user-images.githubusercontent.com/19601248/177232757-419b1327-7d9c-4e62-9923-b8360c58c48d.png">

postman 输出：

<img width="1164" alt="image" src="https://user-images.githubusercontent.com/19601248/177232799-faa4fd1a-7254-488c-a509-85bad9938e53.png">

## 问题：
我改了/path/plugins/stop.py 脚本的值，为什么 输出还是不变?

**Comment by tzssangglass** [comment]
Can you give the full reproduction steps?
